frontend/src/metabase/core/components/Toggle/Toggle.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import Toggle from "./Toggle";
+
+export default {
+  title: "Core/Toggle",
+  component: Toggle,
+  argTypes: { onChange: { action: "onChange" } },
+};
+
+const Template: ComponentStory<typeof Toggle> = args => {
+  return <Toggle {...args} />;
+};
+
+export const Default = Template.bind({});
```

Comment:
It'd be nice to also add use of the props like `small` or `color` here so we can document and see and understand how this component can shift in usage.

Reply:
They already there - just click on controls in storybook